### PR TITLE
Backup api added

### PR DESCRIPTION
**Pull request by saumanbiswas**

### CODE REVIEW

File: kubernetes/v1beta2/client.proto [hunk 1]
```diff
@@ -934,6 +940,50 @@ message Subject {
...
+      int32 keep_monthly_snapshots = 5;
+      int32 keep_yearly_snapshots = 6;
+      repeated string keep_tags = 7;
+      string retain_host_name = 8;
+      repeated string retain_tags = 9;
+    }
+  }
+
+  message Status {
+    string first_backup_time = 1;
```
Comment (sadlil) [inline kubernetes/v1beta2/client.proto]: We do not use string for time.Time. Fix this and return epoc. time.Time().Unix()

File: kubernetes/v1beta2/client.proto [hunk 2]
```diff
@@ -976,9 +976,9 @@ message Backup {
   }
 
   message Status {
-    string first_backup_time = 1;
-    string last_backup_time = 2;
-    string last_successful_backup_time = 3;
+    int64 first_backup_time = 1;
+    int64 last_backup_time = 2;
+    int64 last_successful_backup_time = 3;
     double last_backup_duration = 4;
```
Comment (TamalSaha) [inline kubernetes/v1beta2/client.proto]: `Duration` is int64 in go. Why do we need double? https://golang.org/src/time/time.go?s=14124:14143#L431

Comment (saumanbiswas) [inline kubernetes/v1beta2/client.proto]: backup.Status.LastBackupDuration = backupEndTime.Sub(backupStartTime).Seconds()
we calculated the duration in seconds. second is in float64.

Comment (TamalSaha) [inline kubernetes/v1beta2/client.proto]: Hm. I think may be we just make it a `d.String()`.

**Pull request merged**